transpiler/.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,18 @@
+name: Handle pull request event
```

Comment:
in my understanding, should be only one `.gihub/workflow` for all repository.
could you please move logic to the root and check another folders (crypto3, ....)

Reply:
removed, logic is already moved